azure.yaml

```diff
@@ -1,3 +1,17 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
 
 name: kernel-memory
+workflows:
+  up:
+    steps:
+      - azd: provision
+hooks:
+  postprovision:
+    windows:
+      shell: pwsh
+      interactive: true
+      run: "Write-Host \"Semantic Kernel is ready at: $env:kmServiceEndpoint"
```

Comment:
```suggestion
      run: "Write-Host \"Kernel Memory is ready at: $env:kmServiceEndpoint"
```

Reply:
also, is the double quote needed?

Reply:
No, I think it was just copilot trying to help there XD.

Reply:
Feel free to use this PR as a reference to create your own wording/output.  I was just trying to demonstrate the idea :).  You can close this PR and try your own version